target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -385,6 +386,9 @@ ramips_setup_macs()
 	wnce2001)
 		wan_mac=$(mtd_get_mac_binary factory 46)
 		;;
+        apac1200)
```

Comment:
You can drop it here. You already have `mtd-mac-address = <&config 0x29>` in your dts, which should set the correct mac address. 

I'm not yet sure about the mac address increment. Here you don't have an increment, but in the dts you increment the base mac address by +3.